plugins/node/opentelemetry-instrumentation-ioredis/test/ioredis.test.ts

```diff
@@ -51,13 +51,13 @@ const CONFIG = {
   port: parseInt(process.env.OPENTELEMETRY_REDIS_PORT || '63790', 10),
 };
 
-const URL = `redis://${CONFIG.host}:${CONFIG.port}`;
+const REDIS_URL = `redis://${CONFIG.host}:${CONFIG.port}`;
```

Comment:
Reviewer note: I could revert this variable name change. I had changed it originally because at one point I had been using the [`URL`](https://developer.mozilla.org/en-US/docs/Web/API/URL) object.

Reply:
It's fine to keep the change imo. 🙂